exercises/practice/darts/darts.awk

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env gawk -f
+
+{
+    distance = sqrt($1 ^ 2 + $2 ^ 2)
+    if (distance <= 1) { print 10 }
+    else if (distance <= 5) { print 5 }
+    else if (distance <= 10) { print 1 }
+    else { print 0 }
+}
```

Comment:
Stub file?

Reply:
Woops. Does the validate command overwrite this? Fixing.

Reply:
It does, then it does a `git checkout` to restore it, so if you haven't committed, it's trashed.

Reply:
Yeah. I ran the validate prior to committing.